src/main/kotlin/team/mozu/dsm/adapter/out/organ/persistence/OrganPersistenceAdapter.kt

```diff
@@ -17,6 +18,12 @@ class OrganPersistenceAdapter(
         return organRepository.findByOrganCode(organCode)?.let { organMapper.toModel(it) }
     }
 
+    override fun findById(organId: UUID): Organ? {
+        return organRepository.findById(organId)
+            .map { organMapper.toModel(it) }
+            .orElse(null)
+    }
```

Comment:
코틀린이기 때문에 map을 사용하여 optional로 감싸기보다는 let을 사용하는게 더 적절할 것 같은데 어떠신가요?
```suggestion
    override fun findById(organId: UUID): Organ? {
        return organRepository.findByIdOrNull(organId)
            ?.let { organMapper.toModel(it) }
    }
```